src/test/java/com/openmrs/migrator/core/services/PDIMergeServiceTest.java

```diff
@@ -23,28 +20,23 @@
 @SpringBootTest
 public class PDIMergeServiceTest {
```

Comment:
We need to rework this test - it isn't really testing anything atm since it is mocking the PDI service.

Also the test class needs to be renamed since we removed the PDI Merge Service.

I'd like to see us run a test PDI transformation and Job and see the that the Service runs successfully. This will be an integration test, and the PDI tool has been included as a dependency with the app so no extra setup should be required to run this test.

Please create a folder structure to separate the unit and integration tests (similar to how we did it in the report module

Reply:
ok sure

Reply:
Done